packages/use-sync-external-store/src/useSyncExternalStoreWithSelector.js

```diff
@@ -9,7 +9,7 @@
 
 import * as React from 'react';
 import is from 'shared/objectIs';
-import {useSyncExternalStore} from 'use-sync-external-store/src/useSyncExternalStore';
+import {useSyncExternalStore} from './useSyncExternalStore';
```

Comment:
Perhaps a similar change should be done in other places to at least match the style?

Reply:
What other places?

Reply:
Like here: https://github.com/facebook/react/blob/62a5371f551906a49730c7aa64172114972efc3c/packages/use-sync-external-store/with-selector.js#L12

Reply:
These are entrypoints so I'm not sure they should change. This PR is green and tested and I personally don't want make stylistic changes that invalidates all of that unless it's required to get this merged.

Reply:
This should fail IMHO since there are no "./src/*" exports in package.json.

Reply:
Since the src exports will get removed at the publish stage, it might be a good idea to add it back for local development.

Reply:
> This should fail IMHO since there are no "./src/*" exports in package.json.

`react/packages/use-sync-external-store/with-selector.js` is fed to rollup and will not be shipped with the `src/` path. I would suggest checking out the repository that was used for testing to see how files look after bundling.